docs/Field-Mapping.md

```diff
@@ -283,22 +283,30 @@ All payment methods:
     * `clearing_bankbic` ->  `paidFromBIC` 
    
 `INVOICE-KLARNA`:
-  * _Required_ `language` -> custom field `languageTag` of Type String on the CT Payment
-    * (may be moved to the Order, Cart and Customer Objects later)
+  * _Required_ 
+    * `language` -> custom field `languageTag` of Type String on the CT Payment
+      (may be moved to the Order, Cart and Customer Objects later)
+      
+    * `birthday` -> String custom field `birthday` in Payment in _YYYMMDD_ format.
+    * `ip` -> the IP address of the user -> String custom field `ip` in Payment. 
+    * `gender` -> The value is suppressed to first lowercase character -> String custom field `gender` in Payment
+    * `personalid` -> (***Not supported yet***) Personal ID Nr. 
+          Mandatory for Klarna if customers billing address is in certain nordics countries.
+          -> `personalId` custom field of type String.
```

Comment:
To remember about this mandatory field when we want to implement the service support in nordic countries.